docs/paper/dev/api/scheduler.md

```diff
@@ -0,0 +1,197 @@
+﻿---
+slug: /dev/scheduler
+---
+
+# Task Scheduler
+
+The `BukkitScheduler` can be used to schedule your code to be run later or run it repeatedly.
+
+## What is a tick?
+
+Every game runs something called a game loop which essentially executes all the logic of the game over and over,
+a single execution of that loop is called a 'tick'.
+
+In Minecraft's case the amount of ticks per second is 20, or one tick every 50 milliseconds,
+meaning that the game loop is executed 20 times per second. A tick taking more than 50ms to execute is the moment
+when your server starts to fall behind on its work and lag.
+
+### Converting between human units and Minecraft ticks
+
+Every method of the scheduler that takes a delay or period uses ticks as a unit of time.
+
+Converting from human units to ticks and back is as simple as:  
+`ticks = seconds * 20`  
+`seconds = ticks / 20`
+
+You can make your code more readable by using the
+[`TimeUnit`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/concurrent/TimeUnit.html)
+enum, e.g. to convert 5 minutes to ticks and back:  
+`TimeUnit.MINUTES.toSeconds(5) * 20`  
+`TimeUnit.SECONDS.toMinutes(ticks / 20)`
+
+## Obtaining the scheduler
+
+To obtain a scheduler you can use the instance method on the `Server` class, e.g. in your `onEnable` method:
+
+```java
+@Override
+public void onEnable() {
+	BukkitScheduler scheduler = this.getServer().getScheduler();
+}
+```
+
+## Scheduling tasks
+
+Scheduling a task requires you to pass:
+- your plugin's instance,
+- the code to run, either with a `Runnable` or `Consumer<BukkitTask>`,
+- the delay in ticks before the task should run,
+- if you're scheduling a repeating task - the period in ticks between each execution of the task.
```

Comment:
```suggestion
- Your plugin's instance,
- The code to run, either with a `Runnable` or `Consumer<BukkitTask>`,
- The delay in ticks before the task should run for the first time,
- If you're scheduling a repeating task - the period in ticks between each execution of the task.
```

Reply:
Think of it as a long sentence, hence the commas and the period at the end,
will add `for the first time`